build/document-extractor.js

```diff
@@ -110,16 +110,29 @@ function extractSections($) {
  *        data: {....}
  *   }]
  *
- * At the time of writing (Jan 2020), there is only one single special type of
- * section and that's BCD. The idea is we look for a bunch of special sections
- * and if all else fails, just leave it as HTML as is.
+ * Another example is for specification tables. If the input is this:
+ *
+ *   <h2 id="Specifications">Specifications</h2>
+ *   <div class="bc-specs" data-bcd-query="foo.bar.thing">...</div>
+ *
+ * Then, extract the data-bcd-query and return this:
+ *
+ *   [{
+ *     type: "specifications",
+ *     value: {
+ *        query: "foo.bar.thing",
+ *        id: "specifications",
+ *        title: "Specifications",
+ *        spec_urls: {....}
+ *   }]
  */
 function addSections($) {
   const flaws = [];
 
-  const countPotentialBCDDataDivs = $.find("div.bc-data").length;
-  if (countPotentialBCDDataDivs) {
-    /** If there's exactly 1 BCD table the only section to add is something
+  const countPotentialSpecialDivs =
+    $.find("div.bc-data").length + $.find("div.bc-specs").length;
```

Comment:
I haven't tested it but perhaps it would be neater to do:
```suggestion
    $.find("div.bc-data, div.bc-specs").length;
```